releasenotes/notes/51900.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: traffic-management
```

Comment:
```suggestion
area: installation
```

Reply:
updated this and other items in the notes